nonos/api/analysis.py

```diff
@@ -600,13 +600,14 @@ def map(
         return Plotable(dict_plotable)
 
     def save(self, directory="", header_only=False) -> None:
+        operation = "_" if self.operation == "" else self.operation
```

Comment:
This should be equivalent
```suggestion
        operation = self.operation or "_"
```

Reply:
(Though I don't get this change)

Reply:
oh wow indeed but for me it's a mystery why